admin_scripts/shell_script_template

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    ${SCRIPT_NAME} [-hv] [-o[file]] args ...
+#%
+#% DESCRIPTION
+#%    This is a script template
+#%    to start any good shell script.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         ${SCRIPT_NAME} (magenta.dk) 0.0.1
+#-    author          ${NAME}
+#-    copyright       Copyright 2018, Magenta Aps"
+#-    license         GNU General Public License
+#-    email           ${EMAIL}
+#-
+#================================================================
+#  HISTORY
+#     2017/06/10 : danni : Script creation
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
```

Comment:
A question about this, since I see that shell scripts and Python scripts have two different styles of template: can we adopt something that works for both? Like the JSON manifests I've been committing alongside my scripts, for example?

Reply:
Yes definitely. At the time I introduced these I tried to find someting pythonic for the python scripts and something bash-like for the bash scripts. But it could make sense to make them one for both.